src/entrepot.cpp

```diff
@@ -0,0 +1,118 @@
+// // fib.radGap = 34.75;
```

Comment:
Is it worth keeping this file in right now? If it's good to have then we can keep it, but would be nice to keep commented code to a minimum I think? Obviously always going to be some here and there, I do it all the time 😓

Reply:
Actually this file should not be in there, it was just a place for me to copy stuff to that I might want to reuse in the future. I'll save it to a folder that is ignored by git.

Reply:
Removed it. Live in the newest commit.